client/elements/addons/sc-static-page.html

```diff
@@ -0,0 +1,56 @@
+<link rel="import" href="../../bower_components/polymer/polymer.html">
+
+<!--
+  Static page scrolling
+-->
+<dom-module id="sc-static-page">
+  <template>
+  </template>
+
+  <script>
+      class SCStaticPage extends Polymer.mixinBehaviors([Polymer.AppLocalizeBehavior], ReduxMixin(Polymer.Element)) {
+
+          static get properties() {
+              return {
+                  currentId: {
+                      type: String,
+                      value: ''
+                  }
+              }
+          }
+
+          connectedCallback() {
+            super.connectedCallback();
+          }
+
+          ready() {
+              super.ready();
+              this.shadowRoot.querySelector('main').addEventListener('click', () => {
+                  setTimeout(() => {
+                      this.currentId = this._scrollToSection(window.location.hash, this.currentId);
+                  }, 0);
+              });
+              setTimeout(() => {
+                      this.currentId = this._scrollToSection(window.location.hash, this.currentId);
+              }, 1000);
+          }
+
+          _scrollToSection(sectionId, currentId) {
+              if (!sectionId || currentId === sectionId) {
+                return currentId;
+              }
+              const firstSection = this.shadowRoot.querySelector(sectionId);
+              if (firstSection) {
+                  firstSection.scrollIntoView({
+                      behavior: 'instant',
+                      block: 'start',
+                      inline: 'nearest'
+                  });
+              }
+              return sectionId;
+          }
+      }
+
+      customElements.define('sc-static-page', SCStaticPage);
```

Comment:
the convention is to have `'sc-static-page'` defined like this at the top of the script:
```
          static get is() {
              return 'sc-static-page';
          }
```
and to use `SCStaticPage.is` here instead.

Reply:
Here: https://www.polymer-project.org/2.0/docs/devguide/registering-elements is says specifically to do it like this.

>// Associate the new class with an element name
>customElements.define('my-element', MyElement);

Wile at the element that is being extended they use the `get is()`.

Reply:
OK, sounds good! I wonder why they keep two distinct conventions.